app/src/components/game/HintPart.vue

```diff
@@ -70,6 +66,7 @@ const right = () => {
         <li
           v-for="response in sessionStore.session.responses[current]"
           :key="response.type"
+          id="content"
```

Comment:
This seems to be weird since it is an id in a for-loop. Is this just for debugging?

Reply:
True, I missed it while rebasing, this will be deleted